win32ss/gdi/eng/engobjects.h

```diff
@@ -153,6 +153,12 @@ typedef struct _FONTGDI {
   BYTE          OriginalItalic;
   LONG          OriginalWeight;
   BYTE          CharSet;
+
+  LONG          tmHeight;
```

Comment:
I would add a comment saying something along the lines of: "Precomputed font metrics (supplements FreeType metrics)" for example...

Reply:
Added.